src/Product.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Bag1 from "./assets/bag_1.png";
+
+function Product({label, img, title, price}){
+    return(
+        <article>
+            <span>{label}</span>
+            <img src={img} />
+            <p>{title}</p>
+            <h4>€{price}</h4>
```

Comment:
Hier had je €{price},- kunnen zetten. In je App.js dan alleen het getal van de prijs